packages/eas-cli/package.json

```diff
@@ -70,32 +70,31 @@
     "wrap-ansi": "7.0.0"
   },
   "devDependencies": {
-    "@graphql-codegen/cli": "^1.19.2",
-    "@graphql-codegen/introspection": "^1.18.0",
-    "@graphql-codegen/typescript": "^1.17.11",
-    "@graphql-codegen/typescript-operations": "^1.17.10",
+    "@graphql-codegen/cli": "2.3.1",
```

Comment:
grapqhl-codegen v2 introduced preResolving types by default, and a bunch of other stuff that affects the codegeneration. If you update this dep, then you'd want to rerun codegen as well, as this was quite a significant release of graphql-codegen with lots of changes to the codegeneration.